en/orm/validation.rst

```diff
@@ -438,6 +438,11 @@ two rules::
     $rules->add($rules->validCount('tags', 3, '>=', 'You must have at least 3 tags'));
     $rules->add($rules->validCount('tags', 5, '<=', 'You must have at most 5 tags'));
 
+.. note::
+
+   When comparing via the ``<`` or ``<=`` operators, ``validCount()`` will return
+   ``false`` if you do not supply at least an empty list of - say - tags.
+
```

Comment:
It returns false if the property is not countable, regardless of the operator used, so you could probably remove the first part of the sentence.